### PR TITLE
Handle cancel errors + corrections

### PR DESCRIPTION
**Pull request by ccnlui**
## Context
Handle trade cancel errors and trade corrections messages. Users can now call things like:
```
socket.onCancelErrors((tce) => {
  console.log(tce);
});

socket.onCorrections((tc) => {
  console.log(tc);
});
```
## TODO
- ~Double check test coverage~

## Test results
```
{
  T: 'c',
  Symbol: 'BAC',
  Exchange: 'M',
  OriginalID: 52983525097785,
  OriginalPrice: 44.54,
  OriginalSize: 3817,
  OriginalConditions: [ ' ', '7' ],
  CorrectedID: 52983525098568,
  CorrectedPrice: 44.54,
  CorrectedSize: 297726,
  CorrectedConditions: [ ' ', '7' ],
  Tape: 'A',
  Timestamp: 2021-12-07T19:26:34.561Z
}
{
  T: 'x',
  Symbol: 'RCL',
  ID: 71684334236466,
  Exchange: 'D',
  Price: 73.28,
  Size: 20100,
  CancelErrorAction: 'C',
  Tape: 'A',
  Timestamp: 2021-12-07T17:46:38.024Z
}
```

### CODE REVIEW

File: lib/resources/datav2/entityv2.ts [hunk 1]
```diff
@@ -187,6 +187,86 @@ export interface RawLuld {
...
+	oi: "OriginalID",
+	op: "OriginalPrice",
+	os: "OriginalSize",
+	oc: "OriginalConditions",
+	ci: "CorrectedID",
+	cp: "CorrectedPrice",
+	cs: "CorrectedSize",
+	cc: "CorrectedConditions",
+	z: "Tape",
+	t: "Timestamp",
```
Comment (noramehesz) [inline lib/resources/datav2/entityv2.ts:236]: I think this is only bad formatted on the reviewing view, but please double check these.

**Comment by noramehesz** [comment]
Please also modify the `onStockTrade()` function to return with the new types of data as well.

**Review by noramehesz — APPROVED** [review]
looks good to me

File: lib/resources/datav2/entityv2.ts (hunk 1 above)
Comment (ccnlui) [reply lib/resources/datav2/entityv2.ts:236]: double-checked. Just github things :man_shrugging:

**Pull request merged**